pkg/osquery/runtime/runtime.go

```diff
@@ -524,6 +525,32 @@ func (r *Runner) launchOsqueryInstance() error {
 		return errors.Wrap(err, "could not calculate osquery file paths")
 	}
 
+	// TODO: should also handle windows. probably
```

Comment:
We run windows osquery with `--allow-unsafe` for a bunch of reasons around this. Or possible that we didn't understand how to permission the socket. Not sure.